docs/agent-oriented-development.md

```diff
@@ -1,49 +1,56 @@
+## Decentralisation
 
-This is not to say that facilitators and middlemen have no place in a multi-agent system; rather it is the '_commanding reliance on middlemen_' that MAS rejects.
+Allowing for direct peer-to-peer communication makes Multi-Agent Systems 
+(**MAS**) inherently decentralized. This contrasts conventional systems design, 
+such as that of the client-server model, where clients rely on message relay 
+services provided by an intermediary. That does not imply that facilitators of 
+services and middlemen have no place in a multi-agent system; rather it is the 
+notion of a '_commanding reliance on middlemen_' that is rejected.   
 
-**Division of responsibilities:** In a decentralised system, every agent is equally privileged, and (in principle) should be able to interact with any other agent. The idea is very much aligned with the peer-to-peer paradigm, in which it is the voluntary participation and contribution of the peers that create the infrastructure. Therefore, in a decentralised system, there is no central 'enforcer'. This means all the work that would typically fall under the responsibilities of a central entity must be performed by individual parties in a decentralised system. Blockchain-based cryptocurrencies are a good example of this. A notable characteristic of cryptocurrencies is the absence of central trusted entities (e.g. banks). But this in turn means that most security precautions related to the handling of digital assets and the execution of transactions are the responsibility of individuals.
+**Division of responsibilities:** In a decentralised system every agent is equally privileged and (in principle) able to interact with any other agent. Without a central authority that provides the services needed to mediate agent interactions, hosting these becomes part of the agents' responsibility. An example hereof is the access to a registry of contact addresses that is normally maintained by a service provider. Without any form of government, however, a central database that can be updated at the sole discretion of any individual agent is easily corrupted. A system in which every agent is only responsible for maintaining their own set of records is also problematic, since errors and the asynchronous execution of events inevitably leads to a situation in which agents possess different versions of the record. Instead, agents must share responsibility over the state of records and do so by maintaining a local copy of it and periodically synchronizing this state by requiring consensus to be achieved over it.
```

Comment:
We lost the direct connection to DLT/blockchain on rhs which I think is unfortunate

Reply:
agreed, putting it back in